.circleci/config.yml

```diff
@@ -432,21 +432,26 @@ jobs:
           command: npm run test:reputation:1:anotherChain
           environment:
             NODE_OPTIONS: --max-old-space-size=6144
+
+  reputation-test-non-gnosis-2:
+    <<: *job_common
+    steps:
+      - checkout
+      - <<: *step_restore_cache
+      - setup_remote_docker:
+          version: docker23
+      - <<: *step_pull_solc_docker
+      - <<: *step_setup_global_packages
       - run:
-          name: "Reset chains"
+          name: "Install lsof"
           command: |
             sudo apt-get update
             sudo apt-get install lsof
-            npm run stop:blockchain:client && rm -rf ganache-chain-db*
       - run:
           name: "Running reputation system unit tests"
           command: npm run test:reputation:2:anotherChain
           environment:
             NODE_OPTIONS: --max-old-space-size=6144
-      # - run:
-      #     name: "Running storage consistency smoke tests"
-      #     command: npm run test:contracts:smoke
-
```

Comment:
Yeah, given this is unlikely to come back, could also remove this from package.json, as well as the test files themselves.